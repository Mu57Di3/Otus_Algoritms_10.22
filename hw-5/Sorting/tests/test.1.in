100
911,503,559,544,187,21,655,259,957,670,626,517,308,124,510,792,676,363,415,411,759,278,546,673,763,159,859,512,518,430,951,616,221,830,554,63,951,605,981,825,835,706,550,80,682,400,670,891,347,278,108,541,237,640,7,573,622,731,284,187,516,92,177,836,595,272,733,761,186,844,817,844,119,687,872,463,427,794,629,815,355,102,882,518,534,380,261,160,864,80,174,373,941,168,454,753,15,186,981,550
