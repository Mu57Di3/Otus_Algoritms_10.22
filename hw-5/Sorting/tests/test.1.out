7,15,21,63,80,80,92,102,108,119,124,159,160,168,174,177,186,186,187,187,221,237,259,261,272,278,278,284,308,347,355,363,373,380,400,411,415,427,430,454,463,503,510,512,516,517,518,518,534,541,544,546,550,550,554,559,573,595,605,616,622,626,629,640,655,670,670,673,676,682,687,706,731,733,753,759,761,763,792,794,815,817,825,830,835,836,844,844,859,864,872,882,891,911,941,951,951,957,981,981
